downloads/aws/README.md

```diff
@@ -0,0 +1,81 @@
+Scripted setup to get your aws account ready for Spinnaker
```

Comment:
I'm thinking this readme should live in the quickstart setup? This way users can read this before having to download all the setup files

Reply:
I agree, and I'll move it there after this appears in the downloads section. I want to do this in two stages so that when the commands appear on the site, the download links are also valid and I don't have to quickly update in case I make a mistake 😄.